### DISCUSSION TITLE
How to cut through material into spoil board?

### DISCUSSION THREAD

**Discussion by andrewerrington** [vlachoudis/bCNC#1711] [Q&A] [answered]
Hello!

This might be a dumb question, but I have searched for an answer and came up empty.

I have bCNC running, and I think I understand its basic operation.

My question is this, suppose I have some material 4.2mm thick, but I want to cut a shape out cleanly, so I want to cut 0.3mm into the spoil board for a total depth of 4.5mm. What is the recommended way to do this? Is there a "proper" way, where Z=0 at the surface of the material, the thickness is set to 4.2mm, and somewhere in the cut operation I can specify an additional 0.3mm? Or do I do it the "simple and obvious" way, where I specify the material thickness is 4.5mm, even though I know it is only 4.2mm?

In the cut operation I tried entering a cut depth greater than the thickness of the material, but I got an error because that is not allowed.

Thank you.

**Comment by Harvie** [comment, marked as answer]
Yes, you should just specify thicker material.

"Proper way" is to have your machine mechanicaly precise enough so you don't have to do such thing. But that might be hard on hobby grade machines, that we often use.

You might try this:
Use some big endmill to mill your spoilboard flat, then zero your Z onto that spoilboard and cut from positive Z all the way down to 0, that way you can be sure you will mill exactly to correct depth unless deflection and similar problems cause you troubles.

I also often put cardboard from old boxes under my material, so i don't waste my wasteboard too much ;-)

**Reply by andrewerrington** [reply]
Thanks for the quick reply. I kind of figured that would be the answer. I'm happy to proceed that way, and it means I am not missing anything.

Probably the Cut operation could be improved. When you ask for a cut then if you don't specify the depth you get the material thickness, if you specify the depth and it's less than or equal to the thickness you get the specified depth, and if you specify the depth and it's greater than the thickness you get a warning message, but the specified depth is accepted. Then you could use variables like "thickness+0.5" for example.

Anyway, I'll do it the simple and obvious way. My machine is not super accurate, so just touching the spoil board is not an option. :)

Thank you.

**Reply by Harvie** [reply]
it's just there for safety... i like to live dangerously, so i usualy set it to something like 15cm (i rarely need to cut more than that) and i don't care anymore. just set the desired depth in cutting operation.

> just touching the spoil board is not an option

it is, when you first machine the spoilboard flat using the same inprecise machine. the imprecisions of spoilboard and gantry will mostly cancel each other out :-) but it depends...

**Reply by andrewerrington** [reply]
You set your material thickness to 15cm? Even though it may be much thinner?

I suppose that's one approach.

**Reply by Harvie** [reply]
yes. it is only there to prevent you from creating cut operation which is too deep. i don't need such check, because the cnc is like my life, i like to fuck it up myself :-D

**Reply by daxliniere** [reply]
> Yes, you should just specify thicker material.

That's only true if you set your WCO z-axis to material top, rather than material bottom/top of spoilboard.

The real way to do it is to specify cut depth of -0.2mm (or similar) below profile when setting your toolpaths in your CAM software.
